plugins/utils/mark-radar.nix

```diff
@@ -3,50 +3,72 @@
   config,
   lib,
   ...
-}:
+} @ args:
 with lib; let
   cfg = config.plugins.mark-radar;
   helpers = import ../helpers.nix {inherit lib;};
   defs = import ../plugin-defs.nix {inherit pkgs;};
+  optionWarnings = import ../../lib/option-warnings.nix args;
+  basePluginPath = ["plugins" "mark-radar"];
 in {
-  options.plugins.mark-radar = {
-    enable = mkEnableOption "mark-radar";
+  imports = [
+    (optionWarnings.mkRenamedOption {
+      option = basePluginPath ++ ["highlight_background"];
+      newOption = basePluginPath ++ ["backgroundHighlight"];
+    })
+    (optionWarnings.mkRenamedOption {
+      option = basePluginPath ++ ["background_highlight_group"];
+      newOption = basePluginPath ++ ["backgroundHighlightGroup"];
+    })
+    (optionWarnings.mkRenamedOption {
+      option = basePluginPath ++ ["highlight_group"];
+      newOption = basePluginPath ++ ["highlightGroup"];
+    })
+    (optionWarnings.mkRenamedOption {
+      option = basePluginPath ++ ["set_default_keybinds"];
+      newOption = basePluginPath ++ ["setDefaultMappings"];
+    })
+  ];
```

Comment:
Just add a date in a comment at the top to know when we can delete them

Reply:
Done